src/components/UserSettings/OptionalCheckbox.js

```diff
@@ -0,0 +1,71 @@
+import React from 'react'
+import { Checkbox } from 'patternfly-react'
+import PropTypes from 'prop-types'
+import { msg } from '_/intl'
+import SelectBox from '../SelectBox'
+import style from './style.css'
+
+const ternaryLogicValues = [
+  {
+    id: 1,
+    value: msg.yes(),
```

Comment:
what would you suggest instead to make hot-reloading possible?

Reply:
Hummmm ... if we want to retain this kind of structure (constant id / value pair), it can get difficult.

So, maybe something like (not tested):
```js
const valueFunctionToRuntimeCall = obj => {
  const defineProperties = {}
  for (const [prop, val] of Object.entries(obj)) {
    defineProperties[prop] = {
      enumerable: true,
      ...(val instanceof Function ? { get () { return val() } } : { value: val },
    }
  }
  return Object.defineProperties({}, defineProperties)
}

const ternaryLogicValues = [
  valueFunctionToRuntimeCall({ id: 1, value: msg.yes, realValue: true }),
  valueFunctionToRuntimeCall({ id: 0, value: msg.no, realValue: false }),
  valueFunctionToRuntimeCall({ id: -1, value: msg.notAvailable, realValue: undefined }),
]
```

Reply:
The code got removed for now (Vm settings only) however we have a very similar case in GlobalUserSettings.
There I've moved the structures inside component to ensure the labels will get reloaded.
@sjd78 What do you think?

Reply:
It may work, I'll give it a test.